data/importer.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const {promisify} = require('util');
+const csv = require('csvtojson'); 
+
+const readFileAsync = promisify(fs.readFile);
+
+class Importer {
+
+    import(path) {
+        return readFileAsync(path)
+        .then(() => {
+            csv().fromFile(path).then((jsonObj) => {
+                console.log(jsonObj);
+            })            
+        });
+    }
+
+    importSync(path) {
+        return csv().fromFile(path)
```

Comment:
I suppose, this function should be Sync, therefore usage of async library csvtojson that works on Promises should be avoided.
Consider using fs.readFileSync.